src/translations/components/ServiceDisruptions.ts

```diff
@@ -0,0 +1,18 @@
+import {translation as _} from '../commons';
+
+const ServiceDisruptionsTexts = {
+  header: {
+    title: _('Driftsavvik', 'Service disruptions'),
+  },
+  body: _(
+    'Oversikt over endringer, forsinkelser og avvik finner du på atb.no:',
+    'Get an overview of changes, delays and service disruptions at atb.no (Norwegian only):      ',
```

Comment:
Litt vel mye whitespace her?

```suggestion
    'Get an overview of changes, delays and service disruptions at atb.no (Norwegian only):',
```

Reply:
![image](https://user-images.githubusercontent.com/72508979/159508938-77f147ad-3f83-4480-98cc-cd1a92edff18.png)